locking/locks.go

```diff
@@ -154,6 +157,17 @@ func (c *Client) UnlockFileById(id string, force bool) error {
 	if err := c.cache.RemoveById(id); err != nil {
 		return fmt.Errorf("Error caching unlock information: %v", err)
 	}
+	path := unlockRes.Lock.Path
+	gitRoot, err := git.RootDir()
+	if err != nil {
+		return err
+	}
+	absPath := filepath.Join(gitRoot, path)
```

Comment:
At the time I did this I was testing with my own fork of lfs-test-server since there were no other servers (not even github's own) that were supporting locking. I noticed yesterday as I was working on this that Github locks were starting to work (and could see how they were adding/fixing stuff on the fly as I tested my git-lfs work on Github as well as my fork). I'll review this because my fork may not be adhering to that aspect of the API.